common/source/docs/common-future-wiki-changes.rst

```diff
@@ -107,11 +107,13 @@ The ENABLE column refers to the :ref:`Q_TAILSIT_ENABLE<Q_TAILSIT_ENABLE>` parame
 :ref:`FLIGHT_OPTIONS<FLIGHT_OPTIONS>`   Function
 =====================================   ======================
 7                                       Enable default airspeed EKF fusion for takeoff (Advanced users only)
-8                                       Remove the TRIM_PITCH_CD on the GCS horizon to show flight controller attitude,not plane's
-9                                       Remove the TRIM_PITCH_CD on the OSD horizon to show flight controller attitude,not plane's
+8                                       Remove TRIM_PITCH_CD on the GCS horizon to show pitch relative to AHRS trim
+9                                       Remove TRIM_PITCH_CD on the OSD horizon to show pitch relative to AHRS trim
 10                                      Adjust mid-throttle to be TRIM_THROTTLE in non-auto throttle modes except MANUAL,instead of midway between MAX and MIN stick values (note that the RCx_TRIM value for the throttle channel (x) MUST BE set to center stick value)
 =====================================   ======================
 
+.. note:: Normally, TRIM_PITCH_CD is subtracted from the AHRS pitch so that the artificial horizon shows pitch relative to TRIM_PITCH_CD. In that case, TRIM_PITCH_CD is the pitch required in stabilized modes to maintain altitude at nominal cruise airspeed.
```

Comment:
not exactly correct..
```suggestion
.. note:: Normally, TRIM_PITCH_CD is subtracted from the AHRS pitch so that the artificial horizon shows pitch as if the flight controller was calibrated with aircraft level position set at TRIM_PITCH_CD instead of flat.  This normally results in the artificial horizon indicating 0 pitch when in cruise at desired cruise speed. TRIM_PITCH_CD is the pitch trim that would be required in stabilized modes to maintain altitude at nominal cruise airspeed and throttle, and for most planes is 1-3 degrees positive, depending on the aircraft design (see :ref:`tuning-cruise`).
```